airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry.py

```diff
@@ -173,7 +175,8 @@ def persist_registry_to_json(
         OutputDataFrame: The registry directory manager.
     """
     registry_file_name = f"{registry_name}_registry"
-    registry_json = registry.json()
+    registry_json = registry.json(exclude_none=True)
```

Comment:
I don't know what to do about this, but we keep tripping over this gotcha. Maybe we need a linting rule?

Reply:
Ya its defiantely an annoying default by pandas. Though I dont think a linting rule is a good idea here as there will certainly be times we want null fields to be shown.